Dockerfile.sdk

```diff
@@ -243,6 +247,15 @@ RUN pip3 install install/python/genai_perf-*.whl
 # Install the dependencies needed to run the client examples. These
 # are not needed for building but including them allows this image to
 # be used to run the client examples.
+
+RUN if [ "$PERF_ANALYZER_BUILD" = "ON" ]; then \
+        pip3 install install/python/genai_perf-*.whl; \
+    else \
+        mkdir -p /workspace/install/python && \
+        cp perf_analyzer/genai_perf-*.whl /workspace/install/python/ && \
+        pip3 install /workspace/install/python/genai_perf-*.whl; \
+    fi
+
```

Comment:
This condition confusing me a bit. 
Looks like in the same scenario two `wheel` packages can be available.